File: python/소수만들기.py
def primenumber(number):
    for i in range(2, number):
        if number % i == 0:
            return False
    return True

def solution(nums):
    answer = 0

    for i in range(len(nums)):
        for j in range(i+1, len(nums)):
            for k in range(j+1, len(nums)):
                if primenumber(nums[i]+nums[j]+nums[k]):
                    answer += 1
    return answer

print(solution([1, 2, 3, 4]))

File: Java/소수만들기.java
class Solution {
    public boolean isPrime(int num) {
        if (num == 0 || num == 1) return false;
        if (num == 2) return true;
        for (int i = 2; i < num / 2; i++) {
            if (num % i == 0) return false;
        }
        return true;      
    }
    
    public int solution(int[] nums) {
        int answer = 0;
        for (int i = 0; i < nums.length; i++) {
            for (int j = i + 1; j < nums.length; j++) {
                for (int k = j + 1; k < nums.length; k++) {
                    if (isPrime(nums[i] + nums[j] + nums[k])) answer++;
                }
            }
        }
        return answer;
    }
}
